pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -320,6 +322,7 @@ type ClusterServicePlanList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ClusterServicePlan represents a tier of a ServiceClass.
+// +k8s:openapi-gen=x-kubernetes-print-columns:custom-columns=NAME:.metadata.name,EXTERNAL NAME:.spec.externalName,BROKER:.spec.clusterServiceBrokerName,CLASS:.spec.clusterServiceClassRef.name
```

Comment:
I think the `CLASS` column as it is defined here is probably useful and an improvement over nothing.

With that said I think this is an area where a kubectl plugin could help immensely.

cc @jberkhahn